Week1/Homework/js-exercises/logString.js

```diff
@@ -0,0 +1,14 @@
+"usestrict";
+
+var myString = "Abuzer ALACA";
+console.log(
+  "The value of myString will be my full name because I assigned my name to it."
+);
+console.log(myString);
+
+myString = "There is an apple.";
+
+console.log(
+  "The value of myString will be a sentence which is 'There is an apple.' because I assigned my name to it."
```

Comment:
Nice, but I think you forgot to remove ` because I assigned my name to it` from the console.log